pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -61,6 +61,9 @@ var istioMtlsTransportSocketMatch = &structpb.Struct{
 	},
 }
 
+// SupportedEcdhCurves for upstream TLS configuration.
+var SupportedEcdhCurves = []string{"X25519", "P-256", "secp384r1"}
```

Comment:
first two are Envoy default list. Added the third one.  We do not add default cipher suites/ecdh curves for upstream tls context today - we rely on Envoy defaults.

Wondering is this the right thing to do or should we atleast make this configurable ?
@howardjohn @hzxuzhonghu  WDYT?

Reply:
I am not an expert on this. Curious: there are many other types of curves, why only add these three?

Reply:
first two are current defaults in Envoy and the third one is needed for the issue mentioned.

Reply:
I know this, my question is why not the others